src/main/resources/db/changelog/create/0002_postal_vote_application.sql

```diff
@@ -36,9 +36,12 @@ CREATE TABLE `postal_vote_application`
     `updated_by`                varchar(255) NULL,
     `version`                   bigint       NOT NULL,
     `status`                    varchar(20)  NOT NULL COMMENT 'status of the record, used for soft deletion',
+    `application_status`        varchar(20)  NOT NULL,
+    `signature_waived`          bit(1),
+    `signature_waived_reason`   varchar(250),
     PRIMARY KEY (`application_id`),
     UNIQUE KEY `postal_vote_application_ems_elector_id_unique_idx` (`ems_elector_id`),
-    KEY `postal_vote_application_application_id_status_idx` (`application_id`,`status`),
-    KEY `postal_vote_application_application_id_gss_code_idx` (`application_id`,`gss_code`),
-    KEY `postal_vote_application_gss_code_idx` (`gss_code`)
+    KEY                         `postal_vote_application_application_id_status_idx` (`application_id`,`status`),
```

Comment:
are you planning on deleting database on dev and int?

Reply:
Yes, we do sir

Reply:
this isn't hooked into postal or proxy yet so thinking won't be any issue clearing the DBs?